### PR TITLE
#58 - Frontend Layouts

### LINKED ISSUE #58: Frontend -  Layouts
Currently, each page in our app includes code for the blurred background, footer, and navigation. To simplify the code, we should move this redundant code into the layout components.

### PR DESCRIPTION
**Pull request by AmonDeShir**
Should close #58.

## Admin
image

## Admin - mobile
image
image

## User
image

## User - mobile
image
image

### CODE REVIEW

**Comment by kamilpiech97** [comment]
conflicts

**Comment by kamilpiech97** [comment]
tests

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. It should be here "Dane o profilu" or something like that instead of "Twoje dane"

image

2. Should be translated to "Profil"

image

3. On dashboard I click "Logout" (not Wyloguj) and I see error - maybe just remove this?

image

**Comment by EwelinaSkrzypacz** [comment]
Conflicts

**Comment by AmonDeShir** [comment]
1. fixed 
image
2. fixed
image
3. fixed
image

**Comment by AmonDeShir** [comment]
## Admin
image
## Super Admin
image

**Comment by AmonDeShir** [comment]
Fixed status messages
image

**Review by EwelinaSkrzypacz — APPROVED** [review]

File: resources/views/app.blade.php
Comment (AleksandraKozubal) [inline resources/views/app.blade.php:1]: i suggest to change lang to pl instead of en.

**Review by AleksandraKozubal — COMMENTED** [review]
Logout on email/verify still doesn't work
image

File: resources/views/app.blade.php
Comment (AmonDeShir) [reply resources/views/app.blade.php:1]: done

File: public/favicon.svg
Comment (AleksandraKozubal) [inline public/favicon.svg:1]: Do we use it somewhere?

Comment (AmonDeShir) [reply public/favicon.svg:1]: No, we don't. Deleted.

**Review by AleksandraKozubal — APPROVED** [review]

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Pull request merged**